third_party/purge/helpers/varnish_helper.php

```diff
@@ -30,12 +32,20 @@ function send_purge_request($site_url = NULL, $site_port = NULL)
 			$port = 80;
 		}
 		
+		echo $purge_url; 
+		
 		$ch = curl_init();
+		
 		curl_setopt($ch, CURLOPT_URL, $purge_url);
 		curl_setopt($ch, CURLOPT_PORT , (int)$port);
-		curl_setopt($ch, CURLOPT_CUSTOMREQUEST,'EE_PURGE');
+		curl_setopt($ch, CURLOPT_HTTPHEADER, array( 'Host: '.$_SERVER['SERVER_NAME'] ) );
+		curl_setopt($ch, CURLOPT_CUSTOMREQUEST,'PURGE');
```

Comment:
For us it was more standard. We use other systems to send through purge requests (e.g. google pagespeed) and changing it meant we didn't have to put in dual config in the vcl. Please feel free to change back though if you like or put abstract to settings page?